services/multilang-support/src/utils.py

```diff
@@ -9,10 +9,11 @@
 # Constants
 MAX_TIME = 0.5  # [seconds], used for maximum time allowed
 # for translation generation in `generate_output_tensor()`
+SUPPORTED_LANGS = ["fr", "de", "es", "it", "nl", "ru", "zh", "vi"]
```

Comment:
Each model takes about ~300MB of storage for its pytorch model weight. Here I have 8 models just as proof of concept that we can easily extend the **multilang** aspect of the service.